### PR TITLE
Fix 16 "Low hanging fruit" spelling issues

### PR DESCRIPTION
**Pull request by SeanKilleen**
### Why:

* Supports #23188 
* A follow-up to #23189, as I was asked to create new PRs for any one-off spelling errors I found there.

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Review by skedwards88 — APPROVED** [review]
Thanks for fixing these errors! We'll get this merged down for you.

**Comment by SeanKilleen** [comment]
@skedwards88 it looks like one of the build steps failed due to an authentication issue within that GitHub action.

Let me know if you want me to push a blank commit in order to restart things in the hopes that it was transient.

**Comment by skedwards88** [comment]
Thanks! Our engineering team is working to resolve the credentials error, and our maintainer will merge this PR once they are resolved.

**Pull request merged**
